.github/workflows/cmake.yml

```diff
@@ -0,0 +1,56 @@
+name: C/C++ CI
```

Comment:
This should probably say `C/C++ CI (CMake)` rather than using the same name in two different workflow scripts.